apps/referral/models/referral.py

```diff
@@ -0,0 +1,27 @@
+# Django Imports
+from django.conf import settings
+from django.db import models
+
+
+# isort: off
+
+
+class Referral(models.Model):
+    """Referral model"""
+
+    referrer = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='referrals',
+        on_delete=models.CASCADE,
+    )
+    referred = models.OneToOneField(
+        settings.AUTH_USER_MODEL,
+        related_name='referred_by',
+        on_delete=models.CASCADE,
```

Comment:
I think we should have `on_delete=models.SET_NULL` so that we can show the referral as deactivated or disabled, instead of just vanishing.